### ISSUE TITLE
Ghost service fails to start 

### ISSUE THREAD

**Issue by arthurlogilab** [YunoHost-Apps/ghost_ynh#17]
```
Jan  8 08:32:33 y systemd[1]: Started ghost systemd service.
Jan  8 08:32:33 y systemd[1992]: ghost.service: Failed at step EXEC spawning /usr/bin/node: No such file or directory
Jan  8 08:32:33 y systemd[1]: ghost.service: Main process exited, code=exited, status=203/EXEC
Jan  8 08:32:33 y systemd[1]: ghost.service: Unit entered failed state.
Jan  8 08:32:33 y systemd[1]: ghost.service: Failed with result 'exit-code'.
Jan  8 08:32:33 y systemd[1]: ghost.service: Service hold-off time over, scheduling restart.
Jan  8 08:32:33 y systemd[1]: Stopped ghost systemd service.
Jan  8 08:32:33 y systemd[1]: ghost.service: Start request repeated too quickly.
Jan  8 08:32:33 y systemd[1]: Failed to start ghost systemd service.
Jan  8 08:32:33 y systemd[1]: ghost.service: Unit entered failed state.
Jan  8 08:32:33 y systemd[1]: ghost.service: Failed with result 'exit-code'.
```

**Comment by arthurlogilab** [comment]
symbolic link from node to nodejs seem to fix this : 
```
cp -l /usr/bin/nodejs /usr/bin/node
```

**Comment by tituspijean** [comment]
Most likely fixed by ca3034b49899dfe7991ff962f4bf2b628206907f. Do not hesitate to reopen if needed.
